makeflow/src/makeflow.c

```diff
@@ -678,6 +682,24 @@ static void makeflow_node_complete(struct dag *d, struct dag_node *n, struct bat
 			}
 		}
 
+		if(info->disk_alloc_full) {
```

Comment:
This does not play nicely with the code below if both monitor and disk allocations are activated.